lib/html-import-template-bundler.js

```diff
@@ -28,17 +28,19 @@ export function bundle(cfg) {
       cwd: baseURL.replace(/\\/g, '/')
     })
     .forEach(function(file) {
-      file = path.resolve(baseURL, file);
-      var content = fs.readFileSync(file, {
-        encoding: 'utf8'
-      });
-
-      var $ = whacko.load(content);
-      var name = getCanonicalName(builder, file, 'view').replace(/!view$/g, '');
-
-      $('template').attr('id', name);
-      var template = $.html('template');
-      templates.push(template);
+      if(file != '.') {
```

Comment:
@donald-slagle Is there any reason why you are using `html template loader`? Are you using Polymer components?